app/scripts/controllers/tables.js

```diff
@@ -251,7 +251,7 @@ angular.module('tables', ['stats', 'sql', 'common', 'tableinfo', 'events'])
             // Table Schema
             var tableStmt = 'SELECT column_name, data_type, is_generated, generation_expression ' +
               'FROM information_schema.columns ' +
-              'WHERE table_schema = ? AND table_name = ?';
+              'WHERE AND table_schema = ? AND table_name = ?';
```

Comment:
`WHERE AND`? Is something missing here? :confused:

Reply:
I forget to remove the AND because I had an additional filter before that was not necessary

Reply:
pushed fixup (same commit)